aequilibrae/paths/setup_Assignment.py

```diff
@@ -43,20 +44,30 @@
         Extension(
             "AoN",
             ["AoN.pyx"],
-            extra_compile_args=['/openmp'],
-            extra_link_args=['/openmp'],
+            extra_compile_args=["/openmp", "/O2"],
+            extra_link_args=["/openmp"],
             include_dirs=[np.get_include()],
+            language="c++",
         )
     ]
 else:
     ext_modules = [
         Extension(
             "AoN",
             ["AoN.pyx"],
-            extra_compile_args=['-fopenmp'],
-            extra_link_args=['-fopenmp'],
-            include_dirs=[np.get_include()],
+            extra_compile_args=["-fopenmp", "-std=c++11", "-O3"],  # do we want -Ofast?
+            extra_link_args=["-lparquet", "-fopenmp"],
+            include_dirs=[np.get_include(), pa.get_include()],
+            language="c++",
+            runtime_library_dirs=["/mnt/c/Users/jan.zill/code/aequilibrae/.venv/lib/python3.7/site-packages/pyarrow/"],
```

Comment:
Do we need this line?  Or to replace this with something generic?

Reply:
yes, done